pkgs/by-name/kc/kcl-cli/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kcl-cli";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = "cli";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ZjEMgQukhBGY3LWqsGmLj3lKfLtNNaQugQs0cSLMb80=";
+  };
+
+  vendorHash = "sha256-jmqKMB85HxAlwH7FVjHrLCZQYuAJrguRfzIz1yMypjw=";
+
+  ldflags = [
+    "-X=kcl-lang.io/cli/pkg/version.version=${version}"
```

```suggestion
    "-X=kcl-lang.io/cli/pkg/version.version=${finalAttrs.version}"
```
Think you missed that here.